rs/tests/message_routing/common/common.rs

```diff
@@ -29,14 +30,18 @@ pub async fn start_all_canisters(
         .enumerate()
         .flat_map(|(x, v)| v.iter().enumerate().map(move |(y, v)| (x, y, v)))
     {
-        let input = (&topology, canister_to_subnet_rate, payload_size_bytes);
+        let input = StartArgs {
+            network_topology: topology.clone(),
+            canister_to_subnet_rate,
+            payload_size_bytes,
+        };
         futures.push(async move {
             let _: String = canister
-                .update_("start", candid, input)
+                .update_("start", candid, (input,))
                 .await
                 .unwrap_or_else(|_| {
                     panic!(
-                        "Starting canister_idx={} on subnet_idx={}",
+                        "Starting canister_idx={} on subnet_idx={} failed.",
```

Comment:
This will conflict with my https://github.com/dfinity/ic/pull/2984.

Reply:
Sorry, didn't have a chance to look at it yet.

Reverted (it was copy-pasta from the one XNet test that was updated).